apps/docs/src/common/pages/patterns/add-multiple-things.tsx

```diff
@@ -0,0 +1,99 @@
+import { FC, Fragment, createElement as h } from 'react';
+import { Helmet } from 'react-helmet-async';
+import { PageProps } from '@not-govuk/app-composer';
+import { A, NavigationMenu } from '@not-govuk/components';
+
+const applyAddAnother = require('../../../../assets/images/p-add-another.svg').default;
+
+const Page: FC<PageProps> = ({ location }) => (
+<div className="govuk-grid-row">
+  <Helmet>
+    <title>Patterns - Home Office Design System</title>
+    <meta name="og:article:section" content="Patterns" />
+  </Helmet>
+
+    <div className="govuk-grid-column-one-quarter">
+      <NavigationMenu items={[
+
+        {
+          href: '/patterns/access-a-service',
+          text: 'Access a service'
+        },
+        {
+          href: '/patterns/add-multiple-things',
+          text: 'Add multiple things'
+        },
+        {
+          href: '/patterns/find-information-on-document',
+          text: 'Find information on a document'
+        },
+        {
+          href: '/patterns/get-more-details',
+          text: 'Get more details'
+        },
+        {
+          href: '/patterns/leave-a-service',
+          text: 'Leave a service'
+        },
+        {
+          href: '/patterns/stop-a-service-timing-out',
+          text: 'Stop a service timing out'
+        }
+      ]} />
+    </div>
+
+    <div className="govuk-grid-column-three-quarters">
+      <span className="govuk-caption-xl">Help users to</span>
+      <h1 className="heading-section">Add multiple things</h1>
+      <div style={{marginTop: "-2em"}}>
+      <p className="govuk-caption-m" style={{marginBottom: "2em"}}>Also known as: multiple loops</p>
```

Comment:
Remember this is mostly a transplant of content from the current website. It might not be worth getting bogged down in questions about the actual content right now.

Reply:
I never thought of it that way. I quickly changed the term 'multiple loops' to 'add another' and amended this to mention the pattern is about adding multiple items of the same thing